nansat/mappers/mapper_sentinel1_l1.py

```diff
@@ -29,189 +29,145 @@
 
 from nansat.vrt import VRT
 from nansat.tools import gdal, initial_bearing
+from nansat.warnings import NansatFutureWarning
 from nansat.exceptions import WrongMapperError, NansatReadError
 from nansat.nsr import NSR
 from nansat.node import Node
 
 
 class Mapper(VRT):
-    '''
-        Create VRT with mapping of Sentinel-1 (A and B) stripmap mode (S1A_SM)
-    '''
-
-    def __init__(self, filename, gdalDataset, gdalMetadata,
-                 manifestonly=False, **kwargs):
+    """ Create VRT with mapping of Sentinel-1 (A and B) stripmap mode (S1A_SM)
+
+    Parameters
+    ----------
+    filename : str
+        name of input Sentinel-1 L1 file
+    gdalDataset : None
+    gdalMetadata : None
+    fast : bool
```

Comment:
it is unclear what happens here - can you explain more clearly? This is regarding lon/lat, right? Perhaps say why they need correction..?

Reply:
also, it seems from the code below that bands are not added if fast is True?

Reply:
More comments added to the docstring